wordpress-mobile/WordPress-Android
# Convert list HTML tags to custom tags for processing nested lists

Fixes #4881 by converting `li`, `ul`, and `ol` tags to custom tags (`WPLI`, `WPOL`, `WPUL`) which allows `WPHtmlTagHandler` to correctly process these tags. Previously Android would handle processing them and incorrectly convert ordered lists into bullet lists.

As mentioned in the ticket, this fix is a bit messy but I don't see another option besides `WebView` for properly displaying these lists.

To test:
* View [this comment](https://examplebloge.wordpress.com/2016/12/07/ordered-list/comment-page-1/#comment-12) in the WordPress app
* Verify that the ordered list is properly rendered

cc @theck13 since you were kind enough to report it 😉
